### ISSUE TITLE
It doesn't look like normalizeBasePath was applied to CSS

### ISSUE THREAD

**Issue by hazelybell** [vituum/vituum#31]
It doesn't look like `normalizeBasePath: true` is applying to CSS.

This is breaking my fonts if `base: './'`

```.css
/* use Andika - Regular in .woff format */
@font-face {
    font-family: AndikaW;
    src: url(/src/fonts/andika/web/Andika-Regular.woff2);
}
```
becomes
```.css
/* use Andika - Regular in .woff format */
@font-face {
    font-family: AndikaW;
    src: url(./Andika-Regular-9F1-MuIc.woff2);
}
```

Which is not correct. But it does work properly if I set `base` to an absolute path:

```.css
/* use Andika - Regular in .woff format */
@font-face {
    font-family: AndikaW;
    src: url(https://uofa-cmput404.github.io/slides-xt/assets/Andika-Regular-9F1-MuIc.woff2);
}
```

My vite.config.js:

```.js
import vituum from 'vituum'
import nunjucks from '@vituum/vite-plugin-nunjucks'
import {prexample} from './src/scripts/prexample.js'

export default {
    base: 'https://uofa-cmput404.github.io/slides-xt/',
    // base: './',
    build: {
        minify: false,
        rollupOptions: {
            output: {
                preserveModules: true,
            },
            preserveEntrySignatures: "exports-only",
        }
    },
    plugins: [
        vituum({
            pages: {
                normalizeBasePath: true,
            }
        }),
        nunjucks({
            root: './src',
            filters: {
                prexample
            },
            options: {
                autoescape: true,
            }
        })
    ]
}
```

**Comment by lubomirblazekcz** [comment]
This is actually a correct behaviour of Vite. What path exactly do you need in output?

**Comment by hazelybell** [comment]
The CSS file starts out as 

`/src/styles/custom.css` and becomes `/dist/assets/src/styles/custom-hash.css`

And the font starts out as

`/src/fonts/andika/web/Andika-Regular.woff2` and becomes `./Andika-Regular2.woff2`

so it ends up looking for it in `/assets/src/styles/Andika-Regular2.woff2`

If thats the correct behavior then maybe the stylesheet should be built to `/dist/assets/custom-hash.css` instead of `/dist/assets/src/styles/custom-hash.css` ?

**Comment by lubomirblazekcz** [comment]
Can you please send me repro on stackblitz? I am not able to replicate your problem.

**Comment by hazelybell** [comment]
I've never used stackblitz but I'll try.

**Comment by hazelybell** [comment]
Well, stackblitz immediately said I was out of uploads on the free plan when I tried to add a font file. However, I made a repro here: https://github.com/hazelybell/css-font-bug-repro https://stackblitz.com/~/github.com/hazelybell/css-font-bug-repro

It seems the bug is related to `build.rollupOptions.output.preserveModules: true`, if I remove that setting and `build.rollupOptions.preserveEntrySignature` it creates the css file in the location you seem to expect. The `preserveEntrySignature` is required to be set if I set `preserveModules: true`.
